core/src/avm1/globals/text_format.rs

```diff
@@ -425,17 +424,36 @@ fn set_bullet<'gc>(
     Ok(())
 }
 
-fn display<'gc>(activation: &mut Activation<'_, 'gc>, _text_format: &TextFormat) -> Value<'gc> {
-    avm1_stub!(activation, "TextFormat", "display");
-    Value::Null
+fn display<'gc>(activation: &mut Activation<'_, 'gc>, text_format: &TextFormat) -> Value<'gc> {
+    text_format
+        .display
+        .as_ref()
+        .map_or(Value::Null, |align| match align {
+            crate::html::TextDisplay::Block => istr!("block").into(),
+            crate::html::TextDisplay::Inline => istr!("inline").into(),
+            crate::html::TextDisplay::None => istr!("none").into(),
+        })
 }
 
 fn set_display<'gc>(
     activation: &mut Activation<'_, 'gc>,
-    _text_format: &mut TextFormat,
-    _value: &Value<'gc>,
+    text_format: &mut TextFormat,
+    value: &Value<'gc>,
 ) -> Result<(), Error<'gc>> {
-    avm1_stub!(activation, "TextFormat", "display");
+    if matches!(value, Value::Undefined | Value::Null) {
+        text_format.display = Some(crate::html::TextDisplay::Block);
+        return Ok(());
+    }
+
+    let value = value.coerce_to_string(activation)?;
+    let display = if value.eq(WStr::from_units(b"inline")) {
```

Comment:
this could be a `&value == b"inline"`, I think?

Reply:
you're right, thanks!